packages/connect-common/files/coins.json

```diff
@@ -2226,7 +2155,23 @@
         }
     ],
     "erc20": [],
-    "eth": [],
+    "eth": [
```

Comment:
just please manually remove this eth record. this is of course temporary solution, automation script should be improved, I am sure I will get to it one day.

Reply:
ah, ok i did it for you by excluding them completely using -E param